pkg/apis/core/validation/validation_test.go

```diff
@@ -6535,9 +6535,12 @@ func TestValidatePodDNSConfig(t *testing.T) {
 			expectedError: true,
 		},
 		{
-			desc: "invalid search path",
+			desc: "invalid search path: more than 63 characters in DNS label",
 			dnsConfig: &core.PodDNSConfig{
-				Searches: []string{"custom?"},
```

Comment:
`custom?` is technically valid in DNS. This test was introduced along with all of the other checks on dnsConfig by af7208047a39694f7015c715466ada48fda1312c, so i think it was an example that would fail the previous check but wasn't added because of a problem. `custom?` is the only test that failed with the new checks.